### PR TITLE
Move to GitHub Actions

### PR DESCRIPTION
**Pull request by Richienb**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Node.js ${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: []
```
Comment (sindresorhus) [inline .github/workflows/main.yml]: ⏫

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Node.js ${{ matrix.node-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [14]
```
Comment (sindresorhus) [inline .github/workflows/main.yml]: Vertical list for consistency.

**Comment by sindresorhus** [comment]
I assume you're using some kind of script?

Do you intend to do all the repos? Just curious. https://github.com/search?q=user:sindresorhus+filename:travis.yml&type=code

**Comment by Richienb** [comment]
> I assume you're using some kind of script?

Yes, I bodged together a script that takes repo names and does everything else automatically.

About 99% of your repos don't use complicated things in the build script so I'm planning on covering those.

**Comment by sindresorhus** [comment]
Great! Thanks again. This is saving me from a lot of grief.

**Pull request merged**